native_client/BUILD

```diff
@@ -151,7 +152,7 @@ tf_cc_shared_object(
         "//native_client:rpi3-armv8": LINUX_LINKOPTS,
         "//tensorflow:windows": [],
         "//conditions:default": [],
-    }) + tflite_linkopts(),
+    }) + tflite_linkopts() + ["-lstdc++"],
```

Comment:
Shouldn't `-lstdc++` be added in the iOS case only?

Reply:
fixed